### PR TITLE
Restyling accordion buttons

### PR DESCRIPTION
**Pull request by noahmanger**
Now they will look good at all screen sizes.

Fixes: https://github.com/18F/openFEC/issues/713

### CODE REVIEW

File: static/styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,2587 @@
+@charset "UTF-8";
```
Comment (msecret) [inline static/styles/styles.css]: I think this file isn't supposed to be checked in anymore. @jmcarp would know

Comment (noahmanger) [inline static/styles/styles.css]: Oh great. Hope so.

Comment (jmcarp) [inline static/styles/styles.css]: Right, we don't need to check in compiled assets (styles.css, app.js) anymore.

Comment (noahmanger) [inline static/styles/styles.css]: Great. I added them to my .gitignore. Ok to still merge this in though?

Comment (msecret) [inline static/styles/styles.css]: @noahmanger I think you have to run this to remove the actually file now that its being ignore:

```
git rm --cached <file>
```

**Pull request merged**